engine/apps/alerts/tasks/check_escalation_finished.py

```diff
@@ -114,7 +114,7 @@ def check_escalation_finished_task() -> None:
     """
     from apps.alerts.models import AlertGroup
 
-    now = timezone.now()
+    now = timezone.now() - datetime.timedelta(seconds=30)
```

Comment:
should we make this interval a bit bigger (maybe 30 minutes) just to be on the safe side in case the celery tasks take a bit longer to execute?

Reply:
We may miss something important. Maximum time between creation and failing auditing that I've seen was 12 sec (alert group looked healthy and was escalating in parallel). I'd propose increasing this time maximum to 5 minute, wdyt?